### ISSUE TITLE
Loading Retina images?

### ISSUE THREAD

**Issue by drusus-major** [lokesh/lightbox2#141]
Lightbox2.6 will not load my retina-compliant images when appropriate. Any ideas?

Thanks.

**Comment by lokesh** [comment]
Closing out old issues.

If issue is still relevant, please make a note and I will reopen and investigate. Thanks.

**Comment by drusus-major** [comment]
It's still relevant - never been answered.

Thanks.

**Comment by wasnlosdu** [comment]
This is not caused by lightbox. The problem is that retina.js is only checking for images and since lightbox2 uses "Anchor-Tags", which are not images, it never will check those.

You can add data-rjs to your Anchor tags. So it looks like this:

```html
<a
  href="full/gejus.jpg" 
  data-rjs="true"
  data-lightbox="paintings"   
  title="<em>Gejus</em>. 2013-15<br>oil on canvas<br>60 x 60 inches"
  <img src="thumbs/t_gejus.jpg" width="151" height="150" border="0" alt="Gejus" title="Gejus">
</a>
```

TL;DR => data-rjs="true" on ever \<a\> with an retina image.

**Comment by lokesh** [comment]
Closing out inactive issues.

Looks like the original problem was related to a conflict with `retina.js` script.
